elastic/kibana
# [Synthetics] Fix tls alert in non default space

## Summary

Fixes https://github.com/elastic/kibana/issues/160740

Setup tls default alert in non default space 

### Testing

1. Create few monitors in non default space
2. Make sure status and tls rules are created
3. Try using an expired tls monitor https://expired.badssl.com/ and make sure alert is triggered
